### PR TITLE
'Unable to service recovery request when requested count is bigger th…

### PR DESCRIPTION
**Pull request by LokiYang**
…an configured maximum request count' now is a warning instead of an error.

### CODE REVIEW

File: src/molderl_recovery.erl [hunk 1]
```diff
@@ -91,6 +91,9 @@ handle_info({udp, _Client, IP, Port, <<SessionName:10/binary,SequenceNumber:64/b
             ok = gen_udp:send(State#state.socket, IP, Port, Payload),
             lager:debug("[molderl] Replied recovery request from ~p - reply contains ~p messages", [IP, NumMsgs]);
 
+        {warning, Reason} ->
+            lager:warning("[molderl] Unable to service recovery request from ~p because ~s. Ignoring.", [IP, Reason])
```
Comment (pmembrey) [inline src/molderl_recovery.erl]: Missing semicolon? :)

**Comment by jvirtanen** [comment]
Hi, a fellow [MoldUDP64 developer](https://github.com/jvirtanen/nassau) here. Shouldn't the MoldUDP64 server simply serve as many messages as it can fit into the MTU? This is how I interpret the protocol specification, and that's how at least NASDAQ OMX Nordic used to work. Nassau, my MoldUDP64 client for the JVM, simply caps requested message count to the maximum of 65535.

**Comment by pmembrey** [comment]
@jvirtanen happy to discuss this, but this ticket isn't the best place - can you raise a new issue for me to reply to? :)

**Comment by jvirtanen** [comment]
@pmembrey Yes, I will do so.

**Pull request merged**